plugins/deps-core/src/main/java/dependencies.kt

```diff
@@ -63,8 +63,12 @@ internal fun FormaDependency.hasConfigType(configType: ConfigurationType): Boole
 
 fun deps(vararg names: String): NamedDependency = transitiveDeps(names = *names, transitive = false)
 
-fun transitiveDeps(vararg names: String, transitive: Boolean = true): NamedDependency =
-    NamedDependency(names.toList().map { NameSpec(it, Implementation, transitive) })
+fun platform(vararg names: String): NamedDependency = transitiveDeps(names = *names, transitive = false, platform = true)
+
+fun transitivePlatform(vararg names: String): NamedDependency = transitiveDeps(names = *names, transitive = true, platform = true)
+
+fun transitiveDeps(vararg names: String, transitive: Boolean = true, platform: Boolean = false): NamedDependency =
```

Comment:
I think that code is more java style way, by adding several overload methods for passing value for one argument, cause we don't have named parameters as it was in Java. This solution brings complicity for our API and will take some time to support documentation and improvements.
I think we need to add another type `Platform`, but I don't understand, whats impact we will be get from that.